DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -121,6 +121,7 @@ public void ConfigureServices(IServiceCollection services)
                         options.ViewLocationFormats.Add("/Views/TrackingSystem/Delegates/{1}/{0}.cshtml");
                         options.ViewLocationFormats.Add("/Views/TrackingSystem/CourseSetup/{1}/{0}.cshtml");
                         options.ViewLocationFormats.Add("/Views/SuperAdmin/{1}/{0}.cshtml");
+                        options.ViewLocationFormats.Add("/Views/Support/{0}.cshtml");

```

Comment:
Just need to confirm this won't conflict at all with the slightly different location I added for Resources.

Reply:
I've removed this in favour of yours.